clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -90,4 +96,28 @@
         return names;
     }
 
+
+    /**
+     * Postprocess the configuration so that exponential backoff is disabled if the reconnect backoff
+     * is configured, but the maximum reconnect backoff is not configured.
+     *
+     * @param config                    The config object.
+     * @param parsedValues              The parsedValues as provided to postProcessParsedConfig.
+     * @param reconnectBackoffKey       The name of the configuraiton key for reconnect backoff.
+     * @param reconnectBackoffMaxKey    The name of the configuraiton key for maximum reconnect backoff.
```

Comment:
Since `RECONNECT_BACKOFF_MS_CONFIG` and `RECONNECT_BACKOFF_MAX_MS_CONFIG` are defined in this class too, is it really useful to pass the key and value from every usage? It seems like every one of them passes the same configs.